### PR TITLE
Updates for API agnosticism

### PR DESCRIPTION
**Pull request by MattLusher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
-# The DocuSign eSignature API Postman Collection
+# The DocuSign eSignature API Postman Collections
```
Comment (RajRele) [inline README.md:1]: @MatthewLusher the title still has `eSignature`

File: index.html [hunk 2]
```diff
@@ -1,267 +1,173 @@
...
+      <section class="section">
+        <h2>Step 2: Import into Postman</h2>
+        <div id="shownEnv" class="">
+          <p>Be sure to complete step 1 before clicking the button below. This imports the DocuSign Postman collection and your environment variables into your installation of Postman.</p>
+        </div>
+        <div id="hiddenEnv" class="nodisplay">
+          <p>
+          <div class="postman-run-button"
+            data-postman-action="collection/import"
+            data-postman-var-1="8624faf20177f06bd69f"></div>
```
Comment (RajRele) [inline index.html:125]: This uuID should not change, unless we are uploading a new postman collection

File: index.html [hunk 3]
```diff
@@ -1,267 +1,173 @@
...
+          <li><strong>Integration key:</strong> Any app you create with DocuSign is called an <em>integration.</em> All your integrations are identified in your DocuSign demo account by their <em>integration key,</em> which is a <a href="http://guid.one/guid" target="_blank">GUID</a> issued to your integration when you select <strong>Add App / Integration Key</strong> on the Apps and Keys page. If you don't have any integrations in your demo account yet, select that button and give your integration a name. Use the integration key value, which is automatically generated when you click <strong>OK.</strong></li>
+          <li><strong>Corresponding secret key:</strong> For Authentication Code Grant authentication, your integration needs a secret key as well. Select Add Secret Key on the Apps and Keys page and copy the value; you can only do this when you create the key, as it won't be visible thereafter.</li>
+          <li><strong>Environment:</strong> Choose Demo.</li>
+          <li><strong>API Version:</strong> The Postman collection supports the eSignature REST API versions 2 and 2.1. Version 2.1 is preferred; version 2 will be deprecated in the future.</li>
+        </ul>
```
Comment (RajRele) [inline index.html:111]: This dropdown is eSign specific

Comment (MattLusher) [reply index.html:111]: I figured as much, but I don't know what to change yet or how. I figured I'd look at it again once I can see the new structure.

File: index.html (hunk 2 above)
Comment (gsnavin) [reply index.html:125]: Is there a way to not have this UUID not be specified?

**Comment by RajRele** [comment]
I am closing this PR and taking care of this with this -> https://github.com/docusign/postman-esign-api-collection/pull/28

**Pull request closed without merging**